egs2/aphasiabank/asr1/local/score_per_severity.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
```

Comment:
In `asr.sh`, we have an option to create `local/score.sh` to adopt corpus-specific scoring. Could you help to integrate your local score scripts into that?

FYI, you may check the open_li52 or ml_superb recipe to check how to use `local/score.sh`

Reply:
After some consideration, I think it might be better to let the user perform scoring manually. As noted in the README file, `--nlsyms_txt "local/nlsyms.txt"` is used during training but `--nlsyms_txt none` should be used for scoring. Without modifying asr.sh, it's better for user to call it separately to avoid errors

Reply:
By using `local/score.sh` you have the freedom to choose whether to include nlsyms, so I still think it would be better to incorporate in that way. https://github.com/espnet/espnet/blob/master/egs2/TEMPLATE/asr1/asr.sh#L1692 

But it's up to your choice. Simply adding more instructions to readme is also fine.